src/shared/layout/EventLayout.js

```diff
@@ -29,7 +29,7 @@ export const EventLayout = ({
           <div className="event-img">{event.img}</div>
         </div>
         <div className="grid justify-center">
-          <CustomButton name={button1} />
+          <Button text={button1} />
```

Comment:
1. The buttons aren't aligned to the center. 
2. The width difference between button1 and button2 is quite significant. Can you make them to have a fixed width instead (referring to figma)?
<img width="411" alt="image" src="https://user-images.githubusercontent.com/65900990/218411945-e9b11eb0-9ce9-4e4a-bf2b-630d3b6f7cde.png">

Reply:
Will incorporate this in a new PR!